Extends 'spo list get' command with support for retrieving any default list in a given site (#6447)
pnp/cli-microsoft365: docs/docs/cmd/spo/list/list-get.mdx

: URL of the site where the list to retrieve is located.

`-i, --id [id]`
: ID of the list to retrieve information for. Specify either `id`, `title`,`url` or `default` but not multiple.

`-t, --title [title]`
: Title of the list to retrieve information for. Specify either `id`, `title`,`url` or `default` but not multiple.

`--url [url]`
: Server- or site-relative URL of the list. Specify either `id`, `title`,`url` or `default` but not multiple.

`--default`

> : Set if you want to return the default list from the site. Specify either `id`, `title`, `url` or `default` but not multiple.

Reviewer: ```suggestion
: Set to retrieve the default list from the site. Specify either `id`, `title`, `url`, or `default` but not multiple.
```

## Examples

Get the default document library located in the specified site.

```sh

> m365 spo list get --webUrl https://contoso.sharepoint.com/sites/project-x

Reviewer: ```suggestion
m365 spo list get --webUrl https://contoso.sharepoint.com/sites/project-x --default
```

m365 spo list get --url 'Shared Documents' --webUrl https://contoso.sharepoint.com/sites/project-x
```

Get information about the default (document) located in the specified site.

```sh

> m365 spo list get --webUrl https://contoso.sharepoint.com/sites/project-x --default

Reviewer: Let's not use the same example twice.
